zellij-server/src/tab/mod.rs

```diff
@@ -1370,6 +1371,12 @@ impl Tab {
                 .unwrap();
         }
     }
+    pub fn dump_active_terminal(&mut self, file: Option<String>, client_id: ClientId) {
```

Comment:
Sorry to nitpick, but can we call this `dump_active_terminal_screen`? I think it'll be clearer when you don't have context.